code/_onclick/click.dm

```diff
@@ -19,7 +19,7 @@
 /atom/Click(location, control, params)
 	if(flags_atom & INITIALIZED)
 		SEND_SIGNAL(src, COMSIG_CLICK, location, control, params, usr)
-		usr.ClickOn(src, params)
+		usr.ClickOn(src, params, location)
```

Comment:
```suggestion
		usr.ClickOn(src, location, control, params)
```
Propagate this to all the other stances of `ClickOn(` so we preserve the former proc's argument order.